### PR TITLE
Update to README with new installation instructions

### PR DESCRIPTION
**Pull request by kipk49**
Based off of our findings from Issue #1524, this is an update to the README reflecting the current state of the dependencies.

### CODE REVIEW

**Comment by bam241** [comment]
@kipk49 thx lot for putting this in !

File: README.rst [hunk 1]
```diff
@@ -95,23 +95,39 @@ For detailed instructions on installing dependencies, see `Installing Dependenci
 Quick Cyclus Installation
 *************************
```
Comment (gonuke) [inline README.rst]:
```suggestion
The quickest way to install Cyclus and its dependencies relies on using the `conda-forge` channel and the `conda` package manager within the Anaconda python environment.  The following instructions guide you through that approach.
```

Comment (bam241) [reply README.rst]: Do we really want to advertise 'conda' as the quickest way to install cyclus deps ?!??

#swearing

Comment (gonuke) [reply README.rst]: It is what it is... and it's probably true because conda let's you specify version numbers for each package.

**Comment by kipk49** [comment]
@bam241 I made a new file based off of TEMPLATE.rst that contains a description of the changes I'm trying to implement; do I simply save it and leave it in the news folder?

**Comment by bam241** [comment]
> @bam241 I made a new file based off of TEMPLATE.rst that contains a description of the changes I'm trying to implement; do I simply save it and leave it in the news folder?

Yes !

**Pull request merged**